### ISSUE TITLE
Issues with test data loader

### ISSUE THREAD

**Issue by preetom-saha-arko** [nku-shengzheliu/Pytorch-TransVG#10]
How to evaluate the model on test set? When I run the command "python train.py --resume {saved_model_path} --dataset XXX --test", I am getting "Keyerror : 0". This error generates from the **__getitem__** function of test data loader.

Moreover, **phrase_out** and **word_split** are getting returned from the **__getitem__** function but not getting used in **test_epoch** function (located in train.py). Then what's the point of returning these two?

**Comment by preetom-saha-arko** [comment]
@nku-shengzheliu  Can we have some chat regarding this issue? My email id: arko@cse.buet.ac.bd
